components/automate-ui/src/app/pages/+compliance/+scanner/containers/nodes-list/nodes-list.component.html

```diff
@@ -1,96 +1,102 @@
-      <chef-icon>report_problem</chef-icon> {{ (nodeTotals$ | async)?.unreachable | number }}
-    </span>
-  </chef-option>
-  <chef-option class="filter reachable" value='reachable'>
-    <span class="filter-label">Reachable</span>
-    <span class="filter-total">
-      <chef-icon>check_circle</chef-icon> {{ (nodeTotals$ | async)?.reachable | number }}
-    </span>
-  </chef-option>
-  <chef-option class="filter unknown" value='unknown'>
-    <span class="filter-label">Unknown</span>
-    <span class="filter-total">
-      <chef-icon>help</chef-icon> {{ (nodeTotals$ | async)?.unknown | number }}
-    </span>
-  </chef-option>
-</chef-phat-radio>
+<div *ngIf="(nodeTotals$ | async)?.all > 0">
+  <chef-phat-radio
+    class="nodes-list-status-filters"
+    (change)="onStatusFilterChanged($event)"
+    [value]="statusFilter$ | async">
+    <chef-option class="filter all" value='all'>
+      <span class="filter-label">All</span>
+      <span class="filter-total">
+        <chef-icon>storage</chef-icon> {{ (nodeTotals$ | async)?.all | number }}
+      </span>
+    </chef-option>
+    <chef-option class="filter unreachable" value='unreachable'>
+      <span class="filter-label">Unreachable</span>
+      <span class="filter-total">
+        <chef-icon>report_problem</chef-icon> {{ (nodeTotals$ | async)?.unreachable | number }}
+      </span>
+    </chef-option>
+    <chef-option class="filter reachable" value='reachable'>
+      <span class="filter-label">Reachable</span>
+      <span class="filter-total">
+        <chef-icon>check_circle</chef-icon> {{ (nodeTotals$ | async)?.reachable | number }}
+      </span>
+    </chef-option>
+    <chef-option class="filter unknown" value='unknown'>
+      <span class="filter-label">Unknown</span>
+      <span class="filter-total">
+        <chef-icon>help</chef-icon> {{ (nodeTotals$ | async)?.unknown | number }}
+      </span>
+    </chef-option>
+  </chef-phat-radio>
+</div>
 
-<chef-table class="nodes-list-table" (sort-toggled)="onSortToggled($event)">
-  <chef-thead>
-    <chef-tr>
-      <chef-th>
-        Node
-        <chef-sort-toggle sort="name" [order]="orderFor('name')"></chef-sort-toggle>
-      </chef-th>
-      <chef-th>
-        Platform
-        <chef-sort-toggle sort="platform" [order]="orderFor('platform')"></chef-sort-toggle>
-      </chef-th>
-      <chef-th>
-        Status
-        <chef-sort-toggle sort="status" [order]="orderFor('status')"></chef-sort-toggle>
-      </chef-th>
-      <chef-th>
-        Manager
-        <chef-sort-toggle sort="manager" [order]="orderFor('manager')"></chef-sort-toggle>
-      </chef-th>
-      <chef-th></chef-th>
-    </chef-tr>
-  </chef-thead>
+<chef-toolbar>
+  <app-authorized [allOf]="['/api/v0/compliance/scanner/jobs', 'post']">
+    <div *ngIf="(nodeTotals$ | async)?.all === 0" class="empty-state">
+      <p>Add the first nodes to get started!</p>
+    </div>
+    <div [ngClass]="(nodeTotals$ | async)?.all === 0 ? 'empty-state' : ''">
```

Comment:
🤔 There is no clue what nodeTotals is; would you mind fixing the type? That is, in scanner.state.ts extract an interface (perhaps call it `NodeTotals`) that you can re-use in this class, then you can declare `nodeTotals$` in this class as `Observable<NodeTotals>` instead of `Observable<any>`.
Long story short: whenever you encounter `any` try to get rid of it if you can. 😁

Reply:
@msorens , I have added changes as you suggested for  `type`, please check once.